xbmc/music/ContextMenus.cpp

```diff
@@ -35,6 +36,22 @@
 namespace CONTEXTMENU
 {
 
+static bool IsMusicDbItem(const CFileItem& item)
+{
+  if (URIUtils::IsMusicDb(item.GetPath()))
```

Comment:
Only thing I can find wrong after that is that we need to handle `!item.IsParentFolder()` so that the `...` don't show the context menus.